marshmallow-metaclass.py

```diff
@@ -0,0 +1,82 @@
+from marshmallow import Schema, fields
+
+
+class Meta(type):
+    """The meta class."""
+
+    def __new__(cls, name, bases, attrs):
+        """Custom __new__  method."""
+
+        def __init__(self, **kwargs):
+            """Custom __init__ method."""
+            # get all marshmallow fields
+            self.fields = []
+            for field in dir(self):
+                f = getattr(self, field)
+                if hasattr(f, '__module__') \
+                        and f.__module__ == 'marshmallow.fields':
```

Comment:
Will this still work if I extend a field into some other module?  Say

```python
# myproject/fields.py
from marshmallow import fields

class MyString(fields.String):
    pass

# myproject/models.py
from marshmallow-metaclass import Meta  # assuming this import works
from . import fields

class User(metaclass=Meta):
    name = fields.MyString()
```

Reply:
You can do this now. I added an example inside